modules/cli/pages/cbstats/cbstats-collections.adoc

```diff
@@ -9,49 +8,128 @@ Provides information on collections.
 
 ----
 cbstats host:11210 [common options] collections
+  [ <scope>.<collection> | id <collectionID> ]
 ----
 
 == Description
 
-This command is used to display information on collections.
+Displays information on collections.
 
 == Options
 
-There are no options specific to this command.
+When used without a command-specific option, the `collections` command returns information on all collections for the specified bucket, on the specified node.
+If the `<scope>.<collection>` option is used, the command returns information on the collection so specified.
+If the `id <collectionID>` option is used, the command returns information on the collection whose id is passed as the `<collectionID>` argument.
+
 For common [.cmd]`cbstats` options, see xref:cli:cbstats-intro.adoc[cbstats].
 
 == Examples
 
-The following command retrieves collections-related information on an existing bucket named `testBucket`:
+The following command retrieves all collections-related information for the bucket `travel-sample`, on the specified node:
 
 ----
-/opt/couchbase/bin/cbstats -u Administrator -p password \
--b testBucket localhost:11210 collections
+/opt/couchbase/bin/cbstats localhost:11210 \
+-u Administrator -p password \
+-b travel-sample collections
 ----
 
 The output is as follows:
 
 ----
-collection:0x0:items:         2
-collection:0x8:items:         0
-collection:0x9:items:         0
-collection:0xa:items:         0
-manifest:collection:0x0:name: _default
-manifest:collection:0x8:name: my_collection_in_my_scope_1
-manifest:collection:0x9:name: my_collection_in_my_scope_2
-manifest:collection:0xa:name: my_collection_in_default_scope
-manifest:collections:         4
-manifest:default_exists:      true
-manifest:uid:                 4
+0x0:0x0:disk_size:  19827828
+0x0:0x0:items:      31592
+0x0:0x0:mem_used:   21760360
+0x0:0x0:name:       _default
+0x0:0x0:ops_delete: 0
+0x0:0x0:ops_get:    0
+0x0:0x0:ops_store:  1
+0x0:0x0:scope_name: _default
+0x8:0x9:disk_size:  68762
+0x8:0x9:items:      2
+0x8:0x9:mem_used:   291
+0x8:0x9:name:       MyCollection
+0x8:0x9:ops_delete: 0
+0x8:0x9:ops_get:    2
+0x8:0x9:ops_store:  2
+0x8:0x9:scope_name: MyScope
+manifest_uid:       4
 ----
 
-Note that the output identifies the `_default` collection as 0x0, and indicates that this contains `2` items.
-The output also indicates that three additional scopes exist, which are `my_collection_in_my_scope_1`, `my_collection_in_my_scope_2`, and `my_collection_in_default_scope`.
+Each line of the output presents a data _key_.
+Each key is of the format `scopeID` &#58; `collectionID` &#58; `datumLabel`.
```

Comment:
Similar to a comment above, would it help if the cbstat output was more self describing, here i'm wondering if

"sid_0x8:cid_0x9:label  value"

Is better for the end user, they may intially wonder what a cid and sid is (collection id  and scope id), but that's better than having to infer type by the position on the terminal?

Reply:
I think it's a good idea.